static/skywire-manager-src/src/app/components/vpn/pages/vpn-server-list/vpn-server-list.component.ts

```diff
@@ -0,0 +1,877 @@
+import { Component, OnDestroy } from '@angular/core';
+import { Subscription, Observable } from 'rxjs';
+import { MatDialog } from '@angular/material/dialog';
+import { Router, ActivatedRoute } from '@angular/router';
+import { TranslateService } from '@ngx-translate/core';
+
+import { SortingModes, SortingColumn, DataSorter } from 'src/app/utils/lists/data-sorter';
+import { DataFilterer } from 'src/app/utils/lists/data-filterer';
+import { FilterProperties, FilterFieldTypes, PrintableLabel } from 'src/app/utils/filters';
+import { countriesList } from 'src/app/utils/countries-list';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/skycoin/skywire/pullRequest?prid=7163595)
```suggestion
import { countriesList } from "src/app/utils/countries-list";
```